bin/buckinghamshire/add-parish-categories

```diff
@@ -69,6 +97,19 @@ foreach my $parish (@parishes) {
             }
         );
 
+        $new_contact->set_extra_metadata(group => $contact->{group}) if $contact->{group};
+
+        # Only send reports to the parish
+        $new_contact->set_extra_metadata(prefer_if_multiple => 1);
+
+        if ($contact->{extra_fields}) {
+            foreach my $extra (@{$contact->{extra_fields}}) {
```

Comment:
As it's always a new contact, you could probably have `$new_contact->set_extra_fields(@{$contact->{extra_fields}})` here instead of a loop.